pkg/cli/server/server.go

```diff
@@ -467,7 +467,7 @@ func validateNetworkConfiguration(serverConfig server.Config) error {
 		return errors.Wrap(err, "failed to validate cluster-dns")
 	}
 
-	if (serverConfig.ControlConfig.FlannelBackend != "none" || serverConfig.ControlConfig.DisableNPC == false) && (dualCluster || dualService) {
+	if (serverConfig.ControlConfig.DisableNPC == false) && (dualCluster || dualService) {
```

Comment:
Remove the flannel reference from the error message too.

Reply:
good catch!